mergify_engine/queue/merge_train.py

```diff
@@ -233,35 +277,26 @@ def deserialize(
     ) -> "TrainCar":
         if "initial_embarked_pulls" in data:
             initial_embarked_pulls = [
-                EmbarkedPull(*ep) for ep in data["initial_embarked_pulls"]
+                EmbarkedPull.deserialize(train, ep)
+                for ep in data["initial_embarked_pulls"]
             ]
             still_queued_embarked_pulls = [
-                EmbarkedPull(*ep) for ep in data["still_queued_embarked_pulls"]
+                EmbarkedPull.deserialize(train, ep)
+                for ep in data["still_queued_embarked_pulls"]
             ]
 
         else:
             # old format
             initial_embarked_pulls = [
                 EmbarkedPull(
+                    train,
                     data["user_pull_request_number"],  # type: ignore
                     data["config"],  # type: ignore[typeddict-item]
                     data["queued_at"],  # type: ignore[typeddict-item]
                 )
             ]
             still_queued_embarked_pulls = initial_embarked_pulls.copy()
 
-        # backward compat allow_checks_interruption ->
```

Comment:
is this related?

Reply:
@sileht ?

Reply:
I still see it?

Reply:
@sileht ?

Reply:
https://github.com/Mergifyio/mergify-engine/pull/4236/files#diff-8a28074908845762e6f3c7ec1d353fe5226b27bf71a33c88b3b1437ce7ff757cR142-R151